test/DebugInfo/NonSemantic/Shader200/DebugInfoSubrange.ll

```diff
@@ -10,9 +10,17 @@
 ; CHECK-SPIRV: ExtInstImport [[#EISId:]] "NonSemantic.Shader.DebugInfo.200"
 
 ; CHECK-SPIRV: String [[#LocalVarNameId:]] "A$1$upperbound"
-; CHECK-SPIRV: TypeInt [[#TyInt64Id:]] 64 0
-; CHECK-SPIRV: Constant [[#TyInt64Id]] [[#Constant1Id:]] 1 0
-; CHECK-SPIRV: Constant [[#TyInt64Id]] [[#Constant1000Id:]] 1000 0
+; CHECK-SPIRV-DAG: TypeInt [[#TyInt32Id:]] 32 0
+; CHECK-SPIRV-DAG: TypeInt [[#TyInt64Id:]] 64 0
+; CHECK-NOT: THIS LINE IS USED TO SEPARATE DAGs
```

Comment:
@vmaksimo can probably suggest a more graceful solution for such LIT check

Reply:
The CHECK-NOT is used to make the test stricter.  Without the CHECK-NOT the test could pass if the TypeInt's come after the Constant's since the DAG's could be re-ordered.  

https://llvm.org/docs/CommandGuide/FileCheck.html recommends using CHECK-NOT to separate DAGs:

CHECK-NOT: directives could be mixed with CHECK-DAG: directives to exclude strings between the surrounding CHECK-DAG: directives. As a result, the surrounding CHECK-DAG: directives cannot be reordered, i.e. all occurrences matching CHECK-DAG: before CHECK-NOT: must not fall behind occurrences matching CHECK-DAG: after CHECK-NOT:. For example,

Reply:
I though that it should not happen that Types are defined after the Constants, but I double checked the spec and you're right here - it is possible. Although I doubt that we can ever generate type definition before its actual use. 
I believe we can leave this check as is if you prefer